tools/build_defs/repo/http.bzl

```diff
@@ -230,6 +230,8 @@ def _http_file_impl(ctx):
     return _update_integrity_attr(ctx, _http_file_attrs, download_info)
 
 _HTTP_JAR_BUILD = """\
+load("@rules_java//java:java_import.bzl", "java_import")
```

Comment:
You have to call `str(Label("@rules_java//java:java_import.bzl"))` and emit that into the BUILD file string, otherwise the user of the repo rule needs a dep on `rules_java` using that exact `repo_name`.

Reply:
Ok, I updated this PR to use `str(Label("@rules_java//java:java_import.bzl"))`

Reply:
+1 for fixing this on master first.